File: library/src/main/java/me/proxer/library/api/ProxerErrorCallback.java
package me.proxer.library.api;

/**
 * A callback for a failed request.
 *
 * @author Ruben Gees
 */
public interface ProxerErrorCallback {

    /**
     * Called upon an error for each Request with the {@code exception}.
     */
    void onError(ProxerException exception);
}

File: library/src/main/kotlin/me/proxer/library/api/ProxerErrorCallback.kt
package me.proxer.library.api

/**
 * A callback for a failed request.
 *
 * @author Ruben Gees
 */
interface ProxerErrorCallback {

    /**
     * Called upon an error for each Request with the [exception].
     */
    fun onError(exception: ProxerException)
}
